src/infrastructure/mongo/persistence/badge.persistence.ts

```diff
@@ -11,5 +11,17 @@ class CrowdActionBadgePersistence implements IBadge {
 
     @Prop({ required: true })
     readonly minimumCheckIns: number;
+
+    @Prop()
+    readonly accessibilityOutline?: string;
```

Comment:
This should be called icon. Not accessibilityOutline. This applies to everywhere where accessibilityOutline is used in this PR.